internal/util/tls.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"os"
+)
+
+const DefaultTLSVersion = tls.VersionTLS12
+
+func NewTLSConfig(caPath, certPath, keyPath string) (*tls.Config, error) {
```

Comment:
This could probably go into the `kakfa` package just because we have a very similar piece of code in the `server` package. From a quick glance, they seem to have some tiny differences so it would make sense to follow the convention and move it closer to where it's used because it's specific to that package. https://github.com/cerbos/cerbos/blob/b0904d29cce5dcf68706e74274951473294ba568/internal/server/server.go#L321


FYI the default TLS config use by Cerbos is defined here: https://github.com/cerbos/cerbos/blob/b0904d29cce5dcf68706e74274951473294ba568/internal/util/network.go#L31